### ISSUE TITLE
Full or relative path required on render

### ISSUE THREAD

**Issue by thatcher** [JanSchulz/knitpy#12]
The following raises an exception:

```python
>>> report.render('test.pymd', output='html')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "build\bdist.win-amd64\egg\knitpy\knitpy.py", line 649, in render
WindowsError: [Error 123] The filename, directory name, or volume label syntax is incorrect: ''
```

While this does not:
```python
>>> report.render('./test.pymd', output='html')
```

Thinking render should probably gracefully check current working directory even in the absence of ./

Set up code and environment below:
```
C:\work>python
Python 2.7.8 |Anaconda 2.1.0 (64-bit)| (default, Jul  2 2014, 15:12:11) [MSC v.1500 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
Anaconda is brought to you by Continuum Analytics.
Please check out: http://continuum.io/thanks and https://binstar.org
>>> from knitpy.knitpy import Knitpy
>>> report = Knitpy()
```

**Comment by JanSchulz** [comment]
I can reproduce it here...

**Comment by JanSchulz** [comment]
There is a fix in #13 (hopefully...), if you want to install from github...

**Comment by JanSchulz** [comment]
Merged #13, so this here should be done

**Comment by JanSchulz** [comment]
Thanks for the bugreport!
